.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Your github action
+
+on:
+  push:
+    branches:
+      # NOTE: You may want to limit the trigger branch to be "main" or "master" etc.
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install
+
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: docs-build
```

Comment:
`borales/actions-yarn` seems to be somewhat deprecated.  why not just use `yarn` on cli like we do for unit tests? https://github.com/pokey/cursorless-vscode/blob/715a8e77fb41ffd413f5349a8634061f117ec2db/.github/workflows/test.yml#L21-L25